content/applications/sales/point_of_sale/pos_based_marketing.rst

```diff
@@ -0,0 +1,118 @@
+==================
+Marketing features
+==================
+
+Use your POS system to engage with customers directly by sending them promotional offers via email
+or WhatsApp.
+
+Storing contact details
+=======================
+
+This feature requires your customer's contact details, either their email address or phone number.
+
+- **Email addresses**: automatically collected and saved in POS orders when sending a receipt by
+  email.
+- **Phone numbers**: to store phone numbers when sending receipts on WhatsApp or by SMS,
+
+  #. Go to :menuselection:`Configuration --> Settings` and scroll to the :guilabel:`Bills &
+     Receipts` section;
+  #. Activate the :guilabel:`WhatsApp Enabled` or :guilabel:`SMS Enabled` option(s).
+
+.. image:: pos_based_marketing/sms-whatsapp-enabled.png
+   :alt: setting to enable to store phone numbers when sending receipts
+
+If a customer’s contact information is missing, it will be automatically saved in POS orders when
+the receipt is sent via email, SMS, or WhatsApp.
+
+.. tip::
+   Users can also send standalone marketing messages from the Point of Sale orders form.
```

Comment:
I think the tip is a great input, but readers might need more explanation on "where to find the screenshot view".
I did not use 
```suggestion
   From a POS order form, navigate to the :guilabel:`Contact Info` category under the
   :guilabel:`Extra Info` tab, then click the email icon to send standalone marketing messages.
```